### PR TITLE
[Monitor Ingestion] Code gen updates

### PR DESCRIPTION
**Pull request by pvaneck**
The swagger was updated to mark the `upload` operation as internal. This prevents overloads that are ultimately overriden from showing up in the public API/APIView. With this change, the generated overloads no longer need to be manually deleted.

Also, `black` was run on the codebase.

### CODE REVIEW

**Comment by pvaneck** [comment]
@kristapratico Finally resolving the problem with the generated overloads for overridden methods causing confusion in APIViews. No more manual deletion now.

File: sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_operations/_patch.py [hunk 1]
```diff
@@ -64,12 +64,12 @@ def upload(  # type: ignore[override] # pylint: disable=arguments-renamed, argum
                     content_encoding = "gzip"
                 logs.seek(0)
 
-            super().upload(rule_id, stream=stream_name, body=logs, content_encoding=content_encoding, **kwargs)
+            super()._upload(rule_id, stream=stream_name, body=logs, content_encoding=content_encoding, **kwargs)
```
Comment (kristapratico) [inline sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_operations/_patch.py:67]: Do we still need the type: ignore on L29 or does it work now?

**Review by kristapratico — APPROVED** [review]
So happy to see this! 👍

File: sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_operations/_patch.py (hunk 1 above)
Comment (pvaneck) [reply sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_operations/_patch.py:67]: Ooh, great catch! I can actually remove all the ignore markers on that line for both pylint and mypy.

**Pull request merged**